.github/workflows/run-integration-tests.yml

```diff
@@ -32,7 +35,7 @@ jobs:
       - name: Install project to local Maven repository
         run: mvn -T1C install -DskipTests -Dverification.skip
       - name: Run integration test
-        run: xvfb-run ./integration-tests/checkstyle-10.12.4.sh "${{ runner.temp }}/artifacts"
+        run: xvfb-run ./integration-tests/${{ matrix.integration-test }} "${{ runner.temp }}/artifacts"
```

Comment:
There is another mention of `checkstyle-10.12.4` below, so this will not work :).

Reply:
If it's easy we should fix it, but if it's too involved we can defer.